_scripts/instruction-widget/install.js

```diff
@@ -87,8 +87,6 @@ module.exports = function(context) {
 
       if (context.webserver == "apache") {
         context.package = "python-certbot-apache";
-      } else if (context.webserver == "nginx") {
-        context.certonly = true;
```

Comment:
You should add the previous line back and change this line to:
```
context.package = "python-certbot-nginx";
```
Currently, the built instructions tell you to just install Certbot and then run `certbot --nginx`.